src/frontend/src/lib/api/satellites.api.ts

```diff
@@ -32,6 +33,21 @@ export const listDocs = async ({
 	return actor.list_docs(collection, toListParams(params));
 };
 
+export const setDoc = async ({
```

Comment:
please revert this. as discussed we don't want to implement any data operations in this PR.